inspirehep/modules/workflows/views/callback.py

```diff
@@ -58,37 +59,87 @@ def _get_base_url():
 def continue_workflow_callback():
     """Handle callback to continue a workflow.
 
-    Expects the request data to contain a object ID in the
-    nonce field.
+    Expects the request data to contain an object ID in the
+    nonce field. For a full reference of the results data structure see:
+        http://inspirehep.net/help/admin/bibupload-admin-guide#3.7
+
+    From which we only expect only one result (as only one record can be
+    managed at a time in the holding pen).
     """
+    def _send_error_message(id_object, error_message):
```

Comment:
Why sending emails manually from here? I think it's better to use logger.{error/warning/info} which will end up in Sentry (and email will be sent anyway)